tests/utils.js

```diff
@@ -75,6 +75,29 @@ define([
         }
     }
 
+    function assertDataTreeState(tree) {
```

Comment:
What do you think of renaming this to `assertTreeState` since the provided tree could also be a control tree?